qemu/tests/qemu_guest_agent.py

```diff
@@ -139,10 +207,29 @@ def gagent_install(self, session, vm):
         """
         error_context.context("Try to install 'qemu-guest-agent' package.",
                               logging.info)
-        s, o = session.cmd_status_output(self.gagent_install_cmd)
-        if s:
-            self.test.fail("Could not install qemu-guest-agent package"
-                           " in VM '%s', detail: '%s'" % (vm.name, o))
+        if self.params["os_variant"] == 'rhel8':
+            cmd = self.params["gagent_pkg_check_cmd"]
+            s_check, o_check = session.cmd_status_output(cmd)
+            if s_check == 0:
+                error_context.context("Remove the original guest agent pkg.",
+                                      logging.info)
+                session.cmd("rpm -e %s" % o_check.strip())
+            self.gagent_install_cmd = "rpm -ivh %s" % self.qga_pkg_latest_url
```

Comment:
@xiagao you can use 'rpm -Uvh' directly.

Reply:
if install from fast train to slow train, it will not work.
So I remove pkg first and then install it.